containers/Ideas/ExplorePage.js

```diff
@@ -1,18 +1,30 @@
 import React, { useState, useEffect } from 'react'
 import { View, FlatList } from 'react-native'
 import API from '../../BaseApi'
+import AsyncStorage from '@react-native-async-storage/async-storage'
 import { ExploreListItem } from './ExploreListItem'
 import { styles } from './ExplorePage.styles'
 import { ButtonSignOut } from '../../components/ButtonSignOut'
 import { TextSourceSans } from '../../components/TextSourceSans'
+import {useAuthorization} from '../../containers/Auth/AuthProvider.js'
 
 export const ExplorePage = (props) => {
   const [projects, setProjects] = useState([])
+  const {signOut} = useAuthorization()
 
   const fetchProjects = () => {
-    API.getProjects()
-      .then(response => response && setProjects(response))
-      .catch(error => console.warn(error))
+    AsyncStorage.getItem('authToken')
+      .then((token) => API.getProjects(token))
+      .then((response) => {
+        if(response.statusCode === 200) {
+          setProjects(response.data)
+        } else if (response.statusCode === 401) {
+          console.warn('Unauthorized, wrong login?')
+          signOut()
+        } else {
+          console.warn('fetchProjects returned ' + response.statusCode)
```

Comment:
In case we want to utilize the `catch` below, from within the `then`.
we coud `Promise.reject('fetchProjects returned ' ...)`. I think this way we avoid `Unhandled Promise..` errors. But not entirely sure and not tested.

Reply:
Not sure I get that really.. so instead of the catch a Promise.reject()? I just thought, the catch is redundant anyway, because we already catch in BaseApi, right?

Reply:
i meant if it only console.warn in `then`, then we get `Unhandled Promise...` errors.
so we should instead of `console.warn` in `then`, `Promise.reject` this way it will jump into the `catch` which we would still need. But it is also not super related to this PR, it is a more general question how we want to handle errors with fetching from the API

Reply:
@khamui I tried that out now, so I did 
`        } else {
          Promise.reject(new Error('fetchProjects returned ${response.statusCode}'))
        }
      }).catch(error => console.warn(error))
`
but that actually gives a `Unhandled promise rejection` error. Did I do that wrong maybe? 
If I leave the console.log, there is no promise error, but it just prints the warning..

Reply:
i think you have to `return` it so it arrives in `.catch(error)`

Reply:
:bulb: :see_no_evil: